test/src/e2e_vm_tests/test_programs/abort_control_flow/Forc.toml

```diff
@@ -0,0 +1,9 @@
+[project]
+authors = ["Fuel Labs <contact@fuel.sh>"]
+entry = "main.sw"
+license = "Apache-2.0"
+name = "abort_control_flow"
+
+[dependencies]
+core = { git = "http://github.com/FuelLabs/sway-lib-core" }
+std = { git = "http://github.com/FuelLabs/sway-lib-std" }
```

Comment:
Could you add a branch for these, and the manifest below? Also, could remove unused dependencies (I think core is unused).